src/Framework/FieldsAPI/Concerns/AppendNodes.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Give\Framework\FieldsAPI\Concerns;
+
+use Give\Framework\FieldsAPI\Contracts\Node;
+
+trait AppendNodes {
```

Comment:
So this can be deleted now, right?

Reply:
Ya, I thought I had deleted it already, but I guess I just removed its usage from the `Group` class.

Reply:
Resolved in 5b58566